### ISSUE TITLE
msign should allow for friendlier expiry date formats

### ISSUE THREAD

**Issue by jclulow** [joyent/node-manta#61]
It would be neat if you could present expiry dates in some friendlier formats, e.g.

```
msign -E "2d"
msign -E "next wednesday"
msign -E "teatime"
```

**Comment by konobi** [comment]
There's always shell foo that could be used... 
`$(gdate -d 'next tuesday')`

**Comment by jussisallinen** [comment]
Could these changes be merged eventually?

**Comment by bahamas10** [comment]
https://cr.joyent.us/#/c/4707/

```
$ ./node_modules/.bin/nodeunit test/msign.test.js 

msign.test.js
✔ msign (no arguments)
✔ msign ~~/stor/foo
✔ msign -e <expires> ~~/stor/foo
✔ msign -E 1h ~~/stor/foo
✔ msign -E 1s ~~/stor/foo (good argument)
✔ msign -E 1m ~~/stor/foo (good argument)
✔ msign -E 1h ~~/stor/foo (good argument)
✔ msign -E 1d ~~/stor/foo (good argument)
✔ msign -E 1w ~~/stor/foo (good argument)
✔ msign -E 1y ~~/stor/foo (good argument)
✔ msign -E foo ~~/stor/foo (bad argument)
✔ msign -E and -e together

OK: 26 assertions (7105ms)
```

**Comment by bahamas10** [comment]
available in `manta@5.2.0`
